main.js

```diff
@@ -2,29 +2,175 @@
+    dayOfFuture.innerText = ""; // return content to "" before create a new div with new content inside
+
+    //this method allow create a new div and new content
+    userData.days.forEach((day) => {
+      const newDivItem = document.createElement("div");
+      newDivItem.classList.add("dayoffuture_day");
+      newDivItem.innerHTML = `
+    <span class="dayoffuture-day">${day.datetime}</span> 
+        <img src="./icon/${day.icon}.png" alt="" class="img-dayoffuture_icon" />
+        <div class="dayoffuture-minmax-temperature">
+        <div>
+        <span>Min</span>
+        <span class="dayoffuture-minmax">${convertNext15Days(day.tempmin)}
+        </span>
+        </div>
+        <div>
+        <span>Max</span>
+        <span class="dayoffuture-minmax">${convertNext15Days(
+          day.tempmax
+        )}</span>
+        </div>
+        </div>
+    `;
+
+      dayOfFuture.appendChild(newDivItem); // all of new content will be contain by div dayOfFuture
+    });
+  };
+  // if get fail information will executed this
+  function error() {
+    alert("Can't support for you");
+  }
+  // this method allow we get a information of user
+  navigator.geolocation.getCurrentPosition(success, error);
 }
 
-//this function allow we display information when we have written something to input through create a new object, add a class for that object and add html content in that
-function displayWeather() {
-    data.forEach( data => {
+// this function will get information from api and render to html of today information
+async function displayWeatherToday() {
+  let citySearch = searchInput.value.trim();
+
+  let apiURL = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/${citySearch}?key=579XEPE66XU8KA92CEUPVGEE9&icons1`;
+  // 579XEPE66XU8KA92CEUPVGEE9
+  let data = await fetch(apiURL)
+    .then((response) => response.json())
+    .catch((error) => {
+      alert("Something Went Wrong: Try Checking Your Internet Coneciton");
+    });
+  console.log(data);
+  conditions.innerHTML = data.currentConditions.conditions;
+  feelslike.innerHTML = convertToday(data.currentConditions.feelslike);
+  humidity.innerHTML = data.currentConditions.humidity + " %";
+  visibility.innerHTML = data.currentConditions.visibility + " K/m";
+  sunrise.innerHTML = data.currentConditions.sunrise;
+  sunset.innerHTML = data.currentConditions.sunset;
+  todayCity.innerHTML = data.resolvedAddress;
+  todayTemperature.innerHTML = convertToday(data.currentConditions.temp);
+}
+
+//this function allow we display information of next 15 days when we have written location to input and then create a new object, add a class for that object and add html content in that
+async function displayWeatherNext15days() {
+  let citySearch = searchInput.value.trim();
+
+  let apiURL = `https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline//${citySearch}?key=579XEPE66XU8KA92CEUPVGEE9&icons1`;
```

Comment:
typo issue: double slash
`...services/timeline//${citySearch}..`

Reply:
please define this `apiURL` as a global variable to skip the necessary duplication